### DISCUSSION TITLE
Welcome to DeepSpeech Discussions!

### DISCUSSION THREAD

**Discussion by zh794390558** [PaddlePaddle/PaddleSpeech#572] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by FrankWhh** [comment]
swig_decoders无法安装，Could not find a version that satisfies the requirement swig_decoders/Cannot unpack file /tmp/pip-b0i4_jcs-unpack/simple.htm (downloaded from /tmp/pip-6uizigc4-build, content-type: text/html); cannot detect archive format
